src/python/pants/backend/codegen/thrift/java/java_thrift_library.py

```diff
@@ -57,12 +60,23 @@ def check_value_for_arg(arg, value, values):
     # values impact the outcome of the task.  See JavaThriftLibraryFingerprintStrategy.
     self._compiler = check_value_for_arg('compiler', compiler, self._COMPILERS)
     self._language = language
+
+    deprecated_conditional(
+      lambda: rpc_style is not None,
+      '1.5.0.dev0',
+      'rpc_style', 
+      'Pass rpc_style as compiler_args instead e.g. [ \'--finagle\'] for \'finagle\' rpc_style.' 
```

Comment:
This warning should include the target name, in the form of `target.address.spec`.